### DISCUSSION TITLE
Nested dropdown in Multilist filter

### DISCUSSION THREAD

**Discussion by usfprod** [appbaseio/reactivesearch#1735] [Q&A]
Hi everyone,

I was looking through the [vue appbase documentation](https://docs.appbase.io/docs/reactivesearch/vue/list/MultiList/) for a way to nest a multilist inside a multilist. According to my research this is not possible. The :nestedField property isn't clear if it could help with this kind of feature.

For example something like this:
![image](https://user-images.githubusercontent.com/87011484/124578299-61873b80-de4e-11eb-8a1a-038426f9fc66.png)

Has anyone done this before, if not does anyone have a suggestion for such types of features.

Kind regards,

Youssef Kassem

**Comment by siddharthlatest** [comment]
@usfprod This is an interesting use-case. It can be achieved with the use of [Reactive Component](https://docs.appbase.io/docs/reactivesearch/vue/advanced/ReactiveComponent/). The general approach is that you would pick a design component that provides the desired behavior, e.g. https://vuetifyjs.com/en/components/treeview/#selection-type, and render this inside of ReactiveComponent.

The `nestedField` prop relates to a [nested query in Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/current/nested.html).
